catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -362,23 +257,27 @@ public UpdateResponse update(UpdateStorageRequest streamUpdateRequest)
     //
```

Comment:
I used this comment (and ones similar to it) to distinguish between those methods that were the delegated operations from `CatalogFramework` and those that are support methods, private or shared. It's not necessary, but I find the delineation useful.